### DISCUSSION TITLE
ISO DSD and DSF playback issues

### DISCUSSION THREAD

**Discussion by nUUK25** [beetbox/beets#5237] [Q&A]
Hello everybody,

I'm new here so forgive me if you find my questions trivial !

Currently i'm looking for an Audio streamer (no DAC, and less than 1000 us.) which can playback an ISO DSD file directly and continously track by track for the whole music album. Like Foobar2000 can do as software on a PC. I have looked at: SMSL SD-9, but have no knowledge from users who has first hand experience with it and ISO DSD playback !
I took an ISO DSD file (one music album) and converted into DSF files (with Sonore; ISO2DSD v.7 software) and placed them all in a file folder on a HDD. To my horror i discovered that Foobar2000 software only plays one track and stops ! And likewise the streamer I have borrowed now: Matrix Audio Element S, does exactly the same ! I have to manually start each track which is crap user friendliness ! Element S can't 'see' ISO DSD - hence I converted to DSF.

Can something go wrong in the conversion from ISO DSD to DSF which cause the playback problem ? (In ISO2DSD I chose: Channel mode: Dual, Output mode: Sony DSF, ok Cue sheet, ok Convert DST to DSD, ok Print.)
Is the 'continous playback' problem with DSF files a general problem which can not be changed ?
Has any uses here any experience with ISO DSD playback from any model of Audio Streamer - Especially with continous playback, track by track - I like to hear from you ?

Thanks Lars

**Comment by jackwilsdon** [comment]
This repository is specifically for beets (the music library management software). You're probably best asking these questions on a music forum.
